### PR TITLE
test: check transactions are sorted in reverse chronologically

### PR DESCRIPTION
**Pull request by vindard**
## Description

This is an implicit condition I discovered (internal to Medici) in PR #1044 that I thought we should capture explicitly in our tests.

### CODE REVIEW

File: test/integration/02-user-wallet/02-tx-ordering.spec.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import { Wallets } from "@app"
```
Comment (nicolasburtey) [inline test/integration/02-user-wallet/02-tx-ordering.spec.ts:1]: test in `02-user-wallet` are meant to be part of the "globally consistent tests suite", ie with the check that the double entry ledger is valid (`checkIsBalanced`)

I don't think this test needs to be in 02 folder for the reason we're not checking anything related to the double entry ledger aspect here

Comment (vindard) [reply test/integration/02-user-wallet/02-tx-ordering.spec.ts:1]: Makes sense, but I'm also bootstrapping off of that by the time this test runs the wallet would have had a number of send/receive intraledger/offchain/onchain transactions already recorded to test against.

Is this enough of a reason to keep it here, or should I move this and recreate some of those transactions explicitly?

Comment (nicolasburtey) [reply test/integration/02-user-wallet/02-tx-ordering.spec.ts:1]: you can decide

**Comment by vindard** [comment]
Closing this as stale... we already implicitly test for this

**Pull request closed without merging**